circuits/license/src/error.rs

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use dusk_bytes::Error as BytesError;
+use dusk_plonk::error::Error as PlonkError;
+
+use std::str::ParseBoolError;
+use std::{error, fmt, io};
+
+#[derive(Debug)]
+pub enum Error {
+    BytesError(BytesError),
```

Comment:
Could you suggest how to rename, for example, PlonkError? Should I name it just Plonk? Then it will be quite misleading to have object named Plonk - leading to think that this is a Plonk object. Also, same names are used in transfer contract, should I rename them there too?